osbuild/formats/v1.py

```diff
@@ -223,7 +223,7 @@ def result_for_pipeline(pipeline):
             retval["build"] = result_for_pipeline(build)
             retval["success"] = retval["build"]["success"]
 
-        obj = store and store.get(pipeline.id)
+        obj = store and pipeline.id and store.get(pipeline.id)
```

Comment:
Shouldn't this go into the `metadata` PR as well?

Reply:
`store.get` with `None` worked in prior the integration, but yeah, there are a few cleanup fixes that I want to split out from the WIP commit.